### PR TITLE
7 test coverage up to 80%

### LINKED ISSUE #7: Test coverage up to 80%
- [x] I have read the [Contributing](https://github.com/swapper-org/NodeChain/blob/master/CONTRIBUTING.md) file of the documentation.
- [x] I have searched the [issues](https://github.com/swapper-org/NodeChain/issues) (including closed ones) and believe that this is not a duplicate.
- [x] I am willing to lend a hand to help implement this feature. 

### Feature Description

We need to implement tests for the application that cover at least **80%** of its functionalities.

### LINKED ISSUE #19: Fix Linting Errors
- [x] I have read the [Contributing](https://github.com/swapper-org/NodeChain/blob/master/CONTRIBUTING.md) file of the documentation.
- [x] I have searched the [issues](https://github.com/swapper-org/NodeChain/issues) (including closed ones) and believe that this is not a duplicate.

- **Link to your source / [Gist](https://gist.github.com/)**: 
[Linting NodeChain for first time](https://gist.github.com/swapper-phoenix/c6799cc48364b578d2739d0ea41ceed3)

## Issue

We should fix this linting problems

### PR DESCRIPTION
**Pull request by 0xp3gasus**
# Description

The tests of the RPC and Web sockets methods of the BTC and ETH packages in the corresponding Regtest network developed.

Can not be up to 80% until Web Sockets refactor

Bug solved at opening schemas files

Fixes #7 
Fixes #19 

## Dependencies (if any)

- Connector Python version changed from 3 to 3.7 because of web python module error
- Connector requirements updated to run tests

## Type of change

- [x]  **Bug fix** (non-breaking change which fixes an issue)
- [x]  **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [x] **This change requires a documentation update**

# How Has This Been Tested?

Tests have been developed making requests to regtest nodes and comparing responses with Connector responses. 
Packages used for tests: pytest and coverage

- BTC Tests

BTC tests are developed in file ```Connector/tests/btc/test_btc.py```

BTC Test Coverage v 1 1

- ETH Tests

ETH tests are developed in file ```Connector/tests/eth/test_eth```

ETH Test Coverage v 1 1

Circle CI workflow finished successfully

Connector V1 1 Tests

**Test Configuration**:
Tests run at Circle Ci virtual machine 
Circle CI machine version

# Good practices to consider

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    
+  btctest:
+    machine:
+      image: ubuntu-1604:202007-01
+    environment:
+      STAGE: development
+      PORT: 80
+      SSL_PORT: 443
+      NGINX_CONFIG_PATH: ../../nginx/nginx.conf
+      CERT_PATH: /etc/ssl/certs
```
Comment (phoenix-web3) [inline .circleci/config.yml:42]: This line concerns to SSL nginx reverse proxy. Have you tried to execute the tests with SSL active?

**Review by phoenix-web3 — CHANGES_REQUESTED** [review]
Some changes explaining everything of testing part should be added to `README.md` before merging this PR.

At least, we need to explain how to execute locally lint and tests.

File: Connector/tests/btc/test_btc.py [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+def makeElectrumRequest(method, params):
+    return makeRequest(RPC_ELECTRUM_ENDPOINT, method, params)
+
+
+def mineBlocksToAddress(address, numBlocks=1):
+    return makeBitcoinCoreRequest("generatetoaddress", [numBlocks, address])
+
+
+wallet1Name = "wallet1"
+makeBitcoinCoreRequest("createwallet", [wallet1Name])
```
Comment (phoenix-web3) [inline Connector/tests/btc/test_btc.py]: We should create a conditional to check if the wallet is already created

File: .circleci/config.yml (hunk 1 above)
Comment (0xp3gasus) [reply .circleci/config.yml:42]: No, tests are not executed through the nginx reserve proxy

File: Connector/tests/btc/test_btc.py (hunk 2 above)
Comment (0xp3gasus) [reply Connector/tests/btc/test_btc.py]: Bug fixed

**Review by phoenix-web3 — APPROVED** [review]

**Pull request merged**